homeassistant/components/recorder/util.py

```diff
@@ -206,7 +207,8 @@ def execute_stmt_lambda_element(
                 raise
             time.sleep(QUERY_RETRY_WAIT)
 
-    assert False  # unreachable # pragma: no cover
+    # Unreachable
+    raise RuntimeError  # pragma: no cover
```

Comment:
Can we remove `pragma no cover` here?
```suggestion
    raise AssertionError("Should be unreachable")
```